bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/InstarHandler.java

```diff
@@ -186,26 +186,49 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     public void alarmTriggered(String alarm) {
         ipCameraHandler.logger.debug("Alarm has been triggered:{}", alarm);
-        switch (alarm) {
+        String[] queries = alarm.split("&");
+        // older cameras placed the & for the first query, whilst newer cameras do not.
+        switch (queries[0]) {
             case "/instar?&active=1":// The motion area boxes 1-4
```

Comment:
As you are splitting on "&", I have the feeling these values are no more possible?

Reply:
Thank you for picking this up, will fix tonight.

Reply:
Fixed.